src/scripts/extensions/clipperInject.ts

```diff
@@ -228,7 +226,7 @@ export class ClipperInject extends FrameInjectBase<ClipperInjectOptions> {
 				this.uiCommunicator.callRemoteFunction(Constants.FunctionKeys.escHandler);
 			}
 			if (oldOnKeyDown) {
-				oldOnKeyDown(event);
+				oldOnKeyDown.call(document, event);
```

Comment:
The new compiler was complaining about this line, I _think_ I resolved it the correct way, but having someone double check would be nice ;)